index.html

```diff
@@ -78,13 +79,13 @@
                         <li class="nav__item">
                             <a href="https://flow-launcher.github.io/docs/#/">Docs</a>
                         </li>
-                        <li class="nav__item">
-                            <a href="https://github.com/Flow-Launcher/Flow.Launcher">Github</a>
-                        </li>
 
                         <li class="nav__item">
                             <a href="https://github.com/Flow-Launcher/Flow.Launcher/releases/">Download</a>
```

Comment:
do we want to point to latest?
https://github.com/Flow-Launcher/Flow.Launcher/releases/latest

Reply:
for both download buttons

Reply:
done.